docs/userman/gui/extension/index.md

```diff
@@ -230,7 +230,7 @@ make your way from one example to the next.
 - [Custom static elements](static_element.md)
 - [Custom dynamic elements](dynamic_element/index.md)
 - [Using scalar properties](dynamic_element/scalar_props.md)
-- [Using List of Values](extension_list_of_values.md)
-- [Using tabular data](extension_data.md)
+- [Using List of Values](dynamic_element/list_of_value_props.md)
```

Comment:
Why did you move (and rename?) those files?
They are not related to the dynamicity of the properties. But certainly to the properties' types.